docs/porting-guides/6.x.x.md

```diff
@@ -253,6 +253,26 @@ snmp_settings:
       ipv4_acl: IPV4
 ```
 
+### Configure BGP AS notation to Asdot/Asplain using `bgp_as_notation`
+
+In AVD 5.x, all BGP AS would get converted to `asplain` format in EOS even when it is given in `asdot` format in eos_designs input, since the option to set AS notation was not available in eos_designs.
+
+Starting AVD 6.0.0, BGP AS notation can be set by defining `bgp_as_notation` to `auto`, `asdot` or `asplain` default value is `auto`.
+
+When `bgp_as_notation: auto` (default), AVD takes decision based on the device input AS number:
+    - If the AS number contains a dot (`.`), all other AS numbers get converted to `asdot` notation.
+    - If the AS number does not contain a dot, all other AS numbers get converted to `asplain` notation.
+
+To retain the old behavior, set `bgp_as_notation` to `asplain`.
```

Comment:
```suggestion
It may not be possible to retain the old behavior if you had a mix and match of asdot and asplain notation, which was not rendering what EOS would have rendered anyway.
```